test/logging/example_tlogger_test.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package logging_test
```

Comment:
why different package?

Reply:
So, there's a special case where adding X_test makes it still kinda count as X.
But, I did it because I followed [these instructions](https://blog.golang.org/godoc-documenting-go-code) and specifically [this gob example](https://golang.org/src/encoding/gob/example_test.go).

Reply:
I don't care which way it goes!

Reply:
Ah, I figured out the reason! Examples should only use public interfaces, and by using `package X_test`, it can only use public things.